### DISCUSSION TITLE
Timer

### DISCUSSION THREAD

**Discussion by Windows1Door2Daniel** [FunkinCrew/Funkin#1667] [Ideas]
In many rhythm games there is a 3 seconds timer after closing the Pause menu before the song continues

**Comment by libauror-a** [comment]
Good idea.

**Comment by Windows1Door2Daniel** [comment]
> Good idea.

Thanks

**Comment by aflacc** [comment]
I actually think this would be helpful since if you pause you'd need to be ready

**Comment by inue-kang** [comment]
True, this will be really helpful.
